addons/website_sale_delivery/static/src/js/website_sale_delivery.js

```diff
@@ -88,6 +97,7 @@ publicWidget.registry.websiteSaleDelivery = publicWidget.Widget.extend({
             $amountTax.html(result.new_amount_tax);
             $amountTotal.html(result.new_amount_total);
         }
+        this.updateShippingCost(result.new_amount_total_raw);
```

Comment:
```suggestion
        if (result.new_amount_total_raw !== undefined) {
            this.updateShippingCost(result.new_amount_total_raw);
        }
```

We're in stable, do not update the amounts if the server code wasn't updated and the `new_amount_total_raw` is not given.